app/src/main/java/com/swent/assos/ui/login/SignUpScreen.kt

```diff
@@ -75,37 +83,38 @@ fun SignUpScreen(navigationActions: NavigationActions) {
             val data = hashMapOf("email" to email)
 
             val functions = FirebaseFunctions.getInstance("europe-west6")
-            // Firebase.functions.useEmulator("10.0.2.2", 5001)
+            val config = Config()
 
-            functions.useEmulator("10.0.2.2", 5001)
             // change the region of the function to europe-west6
-
+            if (config.get_all().contains("functions")) {
+              functions.useEmulator("10.0.2.2", 5001)
```

Comment:
Same comment here, [String Resource](https://developer.android.com/guide/topics/resources/string-resource) can be a good idea for the address `10.0.2.2` and port `5001`